python_chess_ai/player/user_input/terminal.py

```diff
@@ -19,15 +25,75 @@ class UserInput(UserInputInterface):
     def __init__(self):
         super().__init__()
 
-    def print_board(self, player_name, board):
-        super().print_board(player_name, board)
-        print(PLAYER_TURN_MESSAGE.format(player_name))
-        print("\n---------------")
-        print(board)
-        print("---------------\n")
+    piece_switcher = {
+        "K": u'\u2654',
+        "Q": u'\u2655',
+        "R": u'\u2656',
+        "B": u'\u2657',
+        "N": u'\u2658',
+        "P": u'\u2659',
+        "k": u'\u265A',
+        "q": u'\u265B',
+        "r": u'\u265C',
+        "b": u'\u265D',
+        "n": u'\u265E',
+        "p": u'\u265F'
+    }
+
+    column_description = {
+        0: " ",
+        1: "A",
+        2: "B",
+        3: "C",
+        4: "D",
+        5: "E",
+        6: "F",
+        7: "G",
+        8: "H"
+    }
 
     def get_move(self, legal_moves):
         super().get_move(legal_moves)
         input_msg = ASK_FOR_MOVE_MESSAGE.format(legal_moves)
         move = Tools.check_legal_input_string(legal_moves,input_msg, WRONG_INPUT_MESSAGE)
         return move
+    
+    def print_board(self, player_name, board):
+        super().print_board(player_name, board)
+        print(PLAYER_TURN_MESSAGE.format(player_name))
+
+        splitted_board = str(board).split("\n")
+        field_number = 0
+        row_counter = 1
+        for row in splitted_board:
+            if row_counter == 1:
+                for i in range(0,9):
+                    print(self.column_description.get(i).center(3), end='')
+                print("\n", end='')
+            splitted_row = row.split(" ")
+            field_number = 0 if field_number %2 is 0 else 1
+            column_counter = 1
+            for chess_field in splitted_row:
+                field_number += 1
+                if column_counter == 1:
+                    colored_field = " " + str(row_counter) + " " + self.create_piece(chess_field, field_number)  
+                elif column_counter == 8:
+                    colored_field = self.create_piece(chess_field, field_number) + " " + str(row_counter)
+                else:
+                    colored_field = self.create_piece(chess_field, field_number)
+                print(colored_field, end='')
+                column_counter += 1
+            print("\n", end='')
+            field_number += 1
+            row_counter += 1
+        for i in range(0,9):
+            print(self.column_description.get(i).center(3), end='')
+        print("\n")
```

Comment:
Neuer Ansatz:
1. Nimm statt board str(board) board.fen()
2. Bilde dir eine zweidimensionale Matrix mit den Schachfiguren ungefähr so (in ner ausgelagerten Funktion):
```
board_fen = board.fen().split(" ")[0]
board_matrix = []

for row in board_fen.split("/"):
    line = []
    for character in row:
        if character.isdigit():
            for empty in range(int(character)):
                line.append(" ")
        else: 
            line.append(character)
    board_matrix.append(line)
```
3. Dann mach ne doppelte for Schleife für Spalten und Reihen. Vor und nach der inneren Schleife printest du die Zahl die die Reihe angibt. In der inneren Schleife erstellst du dann das Feld und printest das. Und außerhalb der äußeren Schleife logischerweise dann das "A B C" gedöns